nengo_dl/version.py

```diff
@@ -1,24 +1,29 @@
-"""
-We use semantic versioning (see http://semver.org/).
+# Automatically generated by nengo-bones, do not edit this file directly
+
+"""NengoDL version information.
+
+We use semantic versioning (see http://semver.org/)
 and conform to PEP440 (see https://www.python.org/dev/peps/pep-0440/).
-'.devN' will be added to the version unless the code base represents
+'.dev0' will be added to the version unless the code base represents
 a release version. Release versions are git tagged with the version.
-
-We avoid the use of f-strings so that this can be imported in Python < 3.6,
-since it is required by setup.py.
 """
 
-import warnings
+version_info = (3, 4, 1)
 
 name = "nengo-dl"
-version_info = (3, 4, 1)  # (major, minor, patch)
-dev = 0  # set to None for releases
+dev = 0
 
+# use old string formatting, so that this can still run in Python <= 3.5
+# (since this file is parsed in setup.py, before python_requires is applied)
 version = "{v}{dev}".format(
     v=".".join(str(v) for v in version_info),
     dev=(".dev%d" % dev) if dev is not None else "",
 )
 
+copyright = "Copyright (c) 2015-2021 Applied Brain Research"
+
+import warnings  # noqa: E402 pylint: disable=wrong-import-position
```

Comment:
Looks like Black did it automatically, haha. Still, would be good to fix in the template.